web/modules/custom/asset_release/asset_release.behat.inc

```diff
@@ -314,4 +314,32 @@ class AssetReleaseSubContext extends DrupalSubContextBase implements DrupalSubCo
     }
   }
 
+  /**
+   * Checks that a release is marked as the latest one.
+   *
+   * This step is meant for the releases overview page within a solution.
+   *
+   * @param string $release
+   *    The name of the release.
+   *
+   * @Then the :release release should be marked as latest
+   *
+   * @throws \Exception
```

Comment:
Respect the tag order in the docblock. First `@param`, then `@return`, then `@throws`, then all others.